app/mailers/user_mailer.rb

```diff
@@ -0,0 +1,10 @@
+class UserMailer < ApplicationMailer
+ default from: 'codeforestdemo@gmail.com'
+  def newsletter
+    @recipients = User.subscribed_users
+    @recipients.each do |r|
```

Comment:
Sending emails to each user should be done in either controller or service.

Think about sending email in async fashion.